cypress-tests/cypress/support/commands/projectsV2.ts

```diff
@@ -0,0 +1,77 @@
+import { TIMEOUTS } from "../../../config";
+
+export type ProjectIdentifier = {
+  slug: string;
+  namespace?: string;
+};
+
+interface NewProjectV2Props extends ProjectIdentifier {
+  visibility?: "public" | "private";
+  name: string;
+}
+
+function getUserNamespaceAPIV2(): Cypress.Chainable<string | null> {
```

Comment:
It would be nice if we explained briefly that this uses the API only.


```suggestion
/** Get the namespace of the logged in user from the API. */
function getUserNamespaceAPIV2(): Cypress.Chainable<string | null> {
```